app/queries/Connect.rb

```diff
@@ -0,0 +1,39 @@
+class Connect

```

Comment:
Filename should be all lower case

Reply:
Also, classname should represent its function, so here it should at least include Query somewhere, ConnectQuery would be better.

Reply:
A Query should always return a ActiveRecord::Relation object, to be able to chain multiple queries.
What you are doing here is not really in accordance with the query pattern, it looks more like a service.
So you also should better make a service out of this class, not a query

Reply:
Still upper case filename, still in the queries directory